src/hooks/useAutoFocusInput.ts

```diff
@@ -35,12 +39,12 @@ export default function useAutoFocusInput(): UseAutoFocusInput {
                 setIsScreenTransitionEnded(true);
             }, CONST.ANIMATED_TRANSITION);
             return () => {
+                setIsScreenTransitionEnded(false);
```

Comment:
Maybe a silly question. Why do we need to set this here?

Reply:
@shubham1206agra We set the state to false whenever screen goes out of focus (it's in the cleanup of `useFocusEffect`), in order to trigger cleanup of `useEffect` above which will cancel ongoing task from `InteractionManager` queue.

In short, it is needed to cancel the focus on screen that is taken out-of-view.

Let me know if I made that clear enough 👍

Reply:
Hi @shubham1206agra do we have any updates on this one? Thanks!

Reply:
Let me know when you'll be able to review this @shubham1206agra, thanks a lot!

Reply:
I will review this today.